docs/it/community-e-networking.md

```diff
@@ -0,0 +1,100 @@
+# Community e Networking
+
+Questo capitolo nasce per due motivi: Il primo è quello di sfatare il mito del dev asociale, il secondo è quello di spiegare come fare networking e perché è così importante in un settore come il nostro.
+
+## Asocialità nel mondo Dev
+
+Chi sviluppa software è spesso visto come una persona asociale, tendenzialmente nerd e propensa a passare le sue giornate senza interagire con nessuno. Eppure, il mondo dello sviluppo software è tra i più vivi da un punto di vista di networking e community. Come mai, quindi, questa percezione è così radicata?
+
+### Perché si è propensi a pensare che i dev siano asociali?
+
+Non possiamo negare che esistano sviluppatori o sviluppatrici che non amano interagire con gli altri o stare in mezzo alla gente e che preferiscono passare le loro giornate in solitudine. Questo è vero, ma sarebbe sbagliato pensare che questo sia un tratto distintivo della nostra professione. In ogni settore lavorativo esistono persone che incarnano queste caratteristiche, quindi non è corretto pensare che il dev sia asociale per natura.
+
+Detto questo, è vero che lo sviluppo software è un lavoro che richiede molta concentrazione e che spesso si lavora in solitudine, il che può portare a una certa tendenza all'isolamento. Soprattutto nei primi decenni di vita della nostra professione, quando non esistevano ancora i social network e le community online, era molto più facile per chi facesse questo lavoro isolarsi, il che ha portato alla nascita di una percezione così radicata che spesso anche chi fa parte del settore si stupisce di quante community esistano e di quanto sia facile fare networking.
```

Comment:
Che ne direste di rielaborare? Potrebbe passare l'idea che si debba entrare in "monk mode" durante i primi dieci anni di carriera.

```suggestion
Detto questo, è vero che lo sviluppo software è un lavoro che richiede molta concentrazione e che spesso si lavora in solitudine, il che può portare a una certa tendenza all'isolamento. Agli albori dell'informatica, quando non esistevano ancora i social network e le community online, era molto più facile per chi facesse questo lavoro isolarsi, il che ha portato alla nascita di una percezione così radicata che spesso anche chi fa parte del settore si stupisce di quante community esistano e di quanto sia facile fare networking.
```